docs/0.3-DRAFT/index.md

```diff
@@ -116,20 +192,21 @@ The structure an _RO-Crate_ MUST follow is:
 
 ```
 <RO-Crate root directory>/
-|   ro-crate-metadata.jsonld  # RO-Crate Metadata File MUST be present
-|   ro-crate-preview.html     # RO-Crate Website homepage MAY be present
-|   ro-crate-preview_files/   # MAY be present
-|    | [other RO-Crate Website files]
-|   [payload files and directories]  # 1 or more SHOULD be present
+  |   ro-crate-metadata.jsonld  # _RO-Crate Metadata File_ MUST be present
```

Comment:
This `_italics_` does not work inside a "```" pre block. We would have to rewrite it to use "\`foo\`" on each line or pure HTML :/

Reply:
The  italics can go. In the next update I've moved this to the end with the bit about repo-specific IDs to make the whole thing a bit less dense